packages/styleguide/src/components/ColorPalette/ColorPalette.js

```diff
@@ -108,6 +108,10 @@ const StyledColorPalette = styled.div`
   list-style-type: none;
   transition: background ease-out 200ms;
   font-family: ${props => props.theme.fontFamily};
+
+  &:last-of-type {
```

Comment:
A preco takto? Preco nema kazda paleta proste margin bottom? Najlepsie ten ktory si pouzil?

Reply:
Aha, uz vidim v kitchen sinku ako si to myslel. Ja som to pouzil v dovere takto: http://dovera-design-system.lbx.sk/styleguide/foundation/color-palette. Chces aby to bolo tak ako to je tu v kitchen sinku?

Reply:
no ked si zoberes tie styly do dovery, tak kazda jedna dostane spodny margin.
ked ich ale bez nadpisov narves pod seba, tak budu pekne spolu.

Reply:
Joj ja som blbec. Dobre to teda je! Krastne dokonca!